components/private_channel/rust/ffi/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
```

Comment:
what is this needed for? We shouldn't be generating anything in the source directory

Reply:
- [x] remove `target` and `Makefile`